app/src/main/res/layout/activity_main.xml

```diff
@@ -46,57 +46,19 @@
         android:background="@drawable/sign_in_button"
         android:textColor="#8BC34A"
         android:textStyle="bold"
-        android:text="Profile PAge"
-        android:id="@+id/profile"
+        android:text="Fit Life Social"
+        android:id="@+id/social"
         />
     <Button
-        android:id="@+id/btnTrack"
+        android:id="@+id/fit"
         android:layout_width="128dp"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
         android:background="@drawable/sign_in_button"
         android:textColor="#8BC34A"
         android:textStyle="bold"
-        android:text="Meal Tracker"/>
-    <Button
+        android:text="Fit Life Fitness"/>
 
-        android:layout_width="128dp"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:background="@drawable/sign_in_button"
```

Comment:
This File Change is going to cause the most errors and conflicts have to find away to not have that happen